eventstore/whatsapp_actions.py

```diff
@@ -85,32 +85,37 @@ def handle_event(event):
 
 
 def handle_fallback_event(event):
-    if event.status == "failed":
-        try:
-            df = DeliveryFailure.objects.get(contact_id=event.recipient_id)
-            df.number_of_failures = F("number_of_failures") + 1
-            df.save()
-            df.refresh_from_db()
-
-            if df.number_of_failures >= 5:
-                async_create_flow_start.delay(
-                    extra={
-                        "optout_reason": OptOut.SMS_FAILURE_REASON,
-                        "timestamp": event.timestamp.timestamp(),
-                        "babyloss_subscription": "FALSE",
-                        "delete_info_for_babyloss": "FALSE",
-                        "delete_info_consent": "FALSE",
-                        "source": "System",
-                    },
-                    flow=settings.RAPIDPRO_OPTOUT_FLOW,
-                    urns=[f"whatsapp:{event.recipient_id}"],
+    if event.status != Event.SENT:
+        if event.status == Event.FAILED:
```

Comment:
This nested if is a bit difficult to follow.

I think it will be clearer if we do something like:
```
if event.status == Event.FAILED:
  ...
else if event.status == Event.DELIVERED or event.status == Event.READ:
  ...
```

Reply:
is it possible to do an `elif` without an `else`? because since `Event.SENT` is a [do nothing], I wouldnt need the else

Reply:
Yes, that's supposed to be an `elif` instead of the `else if` that I typed

Reply:
And yes, it is possible to not have an `else` at the end.

Reply:
not really possible, but is it okay to do it